modules/ROOT/pages/design-migrating-ms.adoc

```diff
@@ -0,0 +1,108 @@
+= Migrate to the Version 2 of the Mocking Service
+
+The mocking service lets you test endpoints that are defined in an API specification. The advantage to using the mocking service is that you do not need to implement the API before testing. To test an endpoint, the mocking service simulates calls to its methods.
+
+For example, when you are designing an API specification in API Designer, you can simulate a call to an endpoint’s GET method to see whether the response body for the 200 status code is as you want it. As a further example, if you are evaluating whether you want to implement an API specification that is in Exchange, you can simulate a call to an endpoint’s GET method to find out what the response body looks like.
+
+[IMPORTANT]
+====
+After January 31, 2020, version 1 of the mocking service will no longer send responses to requests. If you are using a third-party tool to send requests, the mocking service will return the HTTP status code 302 and provide a link to this document.
```

Comment:
Lets the service deprecation date Nov 31 and explain that critical users can still access the Mocking service 1.x via passing this header x- (let's add the exact header info from Martin), and on Jan 31, 2020, the Mocking Service will be completely decommissioned.

Reply:
The header is not yet defined, so to avoid confusion to the user at this point I would say "via passing a special header". Then, in Nov we can update the doc again with the specific instructions and the header already defined

Reply:
@mnbattaglia and @harshakirank , the problem is that users will need the header before November 31, so that there is no break in their continuous use of third-party testing tools. So, I'll need to include a date on which I will update this doc with the header.

Reply:
Fair enough. I'll have that by today EOD